components/automate-cli/pkg/verifyserver/services/s3configservice/s3configservice.go

```diff
@@ -0,0 +1,144 @@
+package s3configservice
```

Comment:
coverage is very less, please add the tests to cover the both positive and negative scenarios